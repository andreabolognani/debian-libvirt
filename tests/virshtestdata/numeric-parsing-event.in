echo Test the <timeout> option (numeric option converted to ms)

echo Non-numeric value
event --all --timeout abc

echo Numeric value that is too big to be converted to ms and still
echo fit inside an int
event --all --timeout 2147484

echo Numeric value with invalid suffix
event --all --timeout 42WB

echo Numeric value with valid suffix. Suffixes are not supported for
echo the <timeout> option, so this value is rejected
event --all --timeout 42MB

echo Negative value
event --all --timeout -1

echo Zero. This is not a valid timeout, but the value is parsed
echo correctly
event --all --timeout 0

echo Numeric value. No events will be received and the command will
echo fail after a second, but the value has been parsed correctly
event --all --timeout 1
