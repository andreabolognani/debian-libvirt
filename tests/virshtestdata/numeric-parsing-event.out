Test the <timeout> option (numeric option converted to ms)
Non-numeric value
error: Numeric value 'abc' for <timeout> option is malformed or out of range

Numeric value that is too big to be converted to ms and still
fit inside an int
error: Numeric value '2147484' for <timeout> option is malformed or out of range

Numeric value with invalid suffix
error: Numeric value '42WB' for <timeout> option is malformed or out of range

Numeric value with valid suffix. Suffixes are not supported for
the <timeout> option, so this value is rejected
error: Numeric value '42MB' for <timeout> option is malformed or out of range

Negative value
error: Numeric value '-1' for <timeout> option is malformed or out of range

Zero. This is not a valid timeout, but the value is parsed
correctly
error: Numeric value '0' for <timeout> option is malformed or out of range

Numeric value. No events will be received and the command will
fail after a second, but the value has been parsed correctly
event loop timed out
events received: 0


## Exit code: 1
